### PR TITLE
Add Extension Main Page

### PR DESCRIPTION
**Pull request by songyiYu**

### CODE REVIEW

**Comment by songyiYu** [comment]
Add styled extension main page

File: styledComponents/StyledButton.tsx [hunk 1]
```diff
@@ -82,16 +82,46 @@ const DownloadButton: typeof RoundedButton = styled(RoundedButton as any)`
...
+    max-width: 200px;
+    width: fit-content;
+    padding: 11px 25px;
     height: 40px;
     background-color: ${props => props.theme.primaryColor};
     border-color: ${props => props.theme.colorPalette.neonBlue};
     margin-left: 15px;
 
+    &:hover {
+      background-color: #5147ff;
```
Comment (jayjongcheolpark) [inline styledComponents/StyledButton.tsx]: @songyiYu How about adding this color to theme for consistency ?

Comment (songyiYu) [reply styledComponents/StyledButton.tsx]: I made a change. thanks for your opinion!

Comment (jayjongcheolpark) [reply styledComponents/StyledButton.tsx]: @songyiYu Good job 👍

**Pull request merged**